src/radiation/radiation_dust_system.hpp

```diff
@@ -0,0 +1,876 @@
+// IWYU pragma: private; include "radiation/radiation_system.hpp"
+#ifndef RADIATION_DUST_SYSTEM_HPP_
+#define RADIATION_DUST_SYSTEM_HPP_
+
+#define LARGE 1.e100
+
+#include "radiation/radiation_system.hpp" // IWYU pragma: keep
+
+template <typename problem_t>
+AMREX_GPU_HOST_DEVICE auto RadSystem<problem_t>::DefinePhotoelectricHeatingE1Derivative(amrex::Real const /*temperature*/, amrex::Real const num_density)
+    -> amrex::Real
+{
+	// const double epsilon = 0.05; // default efficiency factor for cold molecular clouds
+	// const double ref_J_ISR = 5.29e-14; // reference value for the ISR in erg cm^3
+	// const double coeff = 1.33e-24;
+	// return coeff * epsilon * num_density / ref_J_ISR; // s^-1
+	return 0.0;
```

Comment:
Can you explain what is going on with the commented-out code in this routine?

Reply:
This is the cgs values from Bate & Keto, Eq. 26. Previously, I commented it out and replaced it with 0.0 as the default. I'll remove those commentted-out code and move it the PE test problem file.